app/controllers/jobseekers/job_applications/breaks_controller.rb

```diff
@@ -37,6 +37,7 @@ def employment_break
   def employment_break_params
     params.require(:jobseekers_job_application_details_break_form)
           .permit(:reason_for_break, :started_on, :ended_on)
+          .merge("started_on(3i)" => "1", "ended_on(3i)" => "1")
```

Comment:
just a question - what is this this `3i` business?

Reply:
1i is the year, 2i is the month and 3i is the day. What we do here is assign the first day of the month to the date. You can check more here:
https://govuk-form-builder.netlify.app/form-elements/date-field/